src/boot/efi/meson.build

```diff
@@ -111,6 +111,7 @@ if have_gnu_efi
 
         compile_args = ['-Wall',
                         '-Wextra',
+                        '-Werror',
```

Comment:
Please provide some way to disable this. New versions of gcc cause code to fail with Werror quite frequently.

Reply:
ack; will likely change this to have Ubuntu CI provide the -werror param instead